### PR TITLE
Added the python-script for Colour detection project

### PR DESCRIPTION
**Pull request by deepshikha007**
# Description

Check this PR for the issue 
Fixes: #500

### CODE REVIEW

**Review by antrikshmisri — CHANGES_REQUESTED** [review]
@deepshikha007 I think you pushed the wrong files. The script , `Readme.md` are empty.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
There is nothing to review here! @deepshikha007 Please check the files

**Comment by deepshikha007** [comment]
Files are now updated you can check @antrikshmisri  and @kaustubhgupta

**Review by antrikshmisri — CHANGES_REQUESTED** [review]
@deepshikha007 Everything seems to work fine. The only problem is that it shows same color name for every color as you can see below.
<br/>
Screenshot (165)
Screenshot (168)

**Comment by deepshikha007** [comment]
@antrikshmisri I corrected that..you can check now.
Screenshot 2021-03-14 at 6 03 31 PM

Screenshot 2021-03-14 at 6 03 19 PM

**Review by antrikshmisri — APPROVED** [review]
@kaustubhgupta @santushtisharma10  LGTM 💯  , what do you guys say?

**Comment by kaustubhgupta** [comment]
> @kaustubhgupta @santushtisharma10 LGTM 💯 , what do you guys say?

I will review it tomorrow 😀

File: Color_detection/Color_detection.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+import cv2
+
+img_path = 'colorpic.jpg'
```
Comment (kaustubhgupta) [inline Color_detection/Color_detection.py:4]:
```suggestion
img_path = './Color_detection/colorpic.jpg'
```

File: Color_detection/Color_detection.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+csv_path = 'colors.csv'
```
Comment (kaustubhgupta) [inline Color_detection/Color_detection.py:5]:
```suggestion
csv_path = './Color_detection/colors.csv'
```

File: Color_detection/Readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+# Colour Detection using Pandas and OpenCV
+
+In this color detection Python project, we are going to build an application through which you can automatically get the name of the color by clicking on them. So for this, we will have a data file that contains the color name and its values. Then we will calculate the distance from each color and find the shortest one.
+
+# Steps Used
+-Importing required package and load the image.
```
Comment (kaustubhgupta) [inline Color_detection/Readme.md:7]: These bullet points didn't rendered.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Color_detection/Color_detection.py (hunk 2 above)
Comment (deepshikha007) [reply Color_detection/Color_detection.py]: done

File: Color_detection/Readme.md (hunk 3 above)
Comment (deepshikha007) [reply Color_detection/Readme.md]: done @kaustubhgupta

File: Color_detection/Readme.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+
+# Colour Detection using Pandas and OpenCV
+
+In this color detection Python project, we are going to build an application through which you can automatically get the name of the color by clicking on them. So for this, we will have a data file that contains the color name and its values. Then we will calculate the distance from each color and find the shortest one.
+
+# Steps Used
+##-Importing required package and load the image.
```
Comment (kaustubhgupta) [inline Color_detection/Readme.md:7]: This is how it's getting rendered:
image

Try this hierarchy:
```
# Steps Used
- Importing ...
- Reading ...
- Set a mouse ...
    - First
    - Second
 - Draw 
 - Calculate
 - Display
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Color_detection/Readme.md (hunk 4 above)
Comment (deepshikha007) [reply Color_detection/Readme.md]: Done @kaustubhgupta

**Review by kaustubhgupta — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

File: Color_detection/Readme.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+Package used - Pandas and cv2 for OpenCV bindings.
+##-Reading the CSV file with pandas.
```
[the remark this answers is not in the document]
Comment (kaustubhgupta) [reply Color_detection/Readme.md]: @deepshikha007 try this suggestion

[the remark this answers is not in the document]
Comment (deepshikha007) [reply Color_detection/Readme.md]: Done

**Comment by deepshikha007** [comment]
Anything else do i need to do in this project @antrikshmisri @kaustubhgupta

**Comment by kaustubhgupta** [comment]
@deepshikha007 This particular PR will be merged soon. You don't need to do anything in this PR but if you wish you can continue contributing to the project in other issues.

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**